packages/doenetml-worker-rust/lib-doenetml-core/src/components/doenet/section.rs

```diff
@@ -31,11 +38,81 @@ pub use component::SectionActions;
 pub use component::SectionAttributes;
 pub use component::SectionProps;
 
+use super::title::Title;
+
 impl PropGetUpdater for SectionProps {
     fn get_updater(&self) -> Box<dyn PropUpdater> {
         match self {
-            SectionProps::Title => Box::new(ElementRefsProp::new_from_last_matching_child("title")),
+            SectionProps::Title => {
+                Box::new(ElementRefsProp::new_from_last_matching_child(Title::NAME))
+            }
             SectionProps::Hidden => SectionAttributes::Hide.get_prop_updater(),
+            SectionProps::RenderedChildren => Box::new(SectionRenderedChildren::new()),
         }
     }
 }
+
+#[derive(Debug)]
+pub struct SectionRenderedChildren {}
+
+impl SectionRenderedChildren {
+    pub fn new() -> Self {
+        SectionRenderedChildren {}
+    }
+}
+
+impl PropUpdater for SectionRenderedChildren {
+    fn data_queries(&self) -> Vec<DataQuery> {
+        let title_local_idx = LocalPropIdx(
+            Section::PROP_NAMES
+                .into_iter()
+                .position(|name| name.eq(&"title"))
+                .unwrap(),
+        );
+        vec![
+            // all non-hidden children except titles
+            DataQuery::FilteredChildren {
+                filters: vec![
```

Comment:
This vec represents a union? I think we should type this out so it is explicitly a `Filter::Union` or `Filter:Any`, so it's not confused with a `Filter:All`

Reply:
Right, another level rather than the plain vector makes a lot of sense. It's actually programmed as a `All` right now.

It's also not clear what to do in the case when some, but not all, props are missing. I just did the easiest algorithm to program (if one prop is missing, treat as though all props are missing), as I don't know if we actually care enough about this case to put more work into it. But, that's not the algorithm one would naively expect.